src/phase_configs/self_tests/Good.yml

```diff
@@ -0,0 +1,2 @@
+Repeat: {^Parameter: {Name: "Repeat", Default: 1}}
+Mode: NoException
```

Comment:
I'd really like to avoid having self-test yamls shipped to production machines. When debugging the first thing you do is start to suspect that test yamls are getting read etc. If we don't hard-code the path does that help?

Reply:
I think so. We can then put the self test YAMLs elsewhere and not install them.